types/aws-lambda/test/iot-authorizer-tests.ts

```diff
@@ -0,0 +1,148 @@
+import {
+    IoTCustomAuthorizerHandler,
+    IoTCustomAuthorizerEvent,
+    IoTCustomAuthorizerResult, IoTProtocolType, IoTProtocolDataHTTP, IoTProtocolDataTLS, IoTProtocolDataMQTT,
+    PolicyDocument, Statement,
```

Comment:
Could you check the formatting here? The prettier check on CI isn't complaining, which is surprising, but it would be good to avoid dinging a future PR.